pkg/manifest/locator.go

```diff
@@ -0,0 +1,190 @@
+package manifest
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+)
+
+// One or more alternative expressions of the location.
+// https://github.com/readium/architecture/tree/master/models/locators#the-location-object
+type Locations struct {
+	Fragments        []string               `json:"fragments,omitempty"`        // Contains one or more fragment in the resource referenced by the [Locator].
```

Comment:
Since you implemented `MarshalJSON()`, I guess the `json:` tags are redundant?

Reply:
Yes, that's true in many places in the toolkit. I left them just for humans to be able to easily tell what the JSON key of something is, but I can remove them if you'd like

Reply:
No that's actually a good idea, I was just a bit confused.

Could you just remove `json:"otherLocations"` as this one is incorrect? Thanks.